src/containers/Footer/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import Footer from "../../components/Footer/index";
+
+const mapStateToProps = state => ({ });
+
+const mapDispatchToProps = dispatch => ({ });
+
+const FooterContainer = (props) => (
+  <Footer {...props} />
+);
+
+export default connect(mapStateToProps, mapDispatchToProps)(FooterContainer);
```

Comment:
Not now, I just prepared a template for future implementation (although we don't actualy have a footer on the designs...). Since it's not in the scope of the taks, should we get rid of it?